docs/app/DocsApp.js

```diff
@@ -10,26 +11,32 @@ import ComponentDoc from './Components/ComponentDoc/ComponentDoc'
 import DocsMenu from './Components/Sidebar/Sidebar'
 import style from './Style'
 
-const { Grid } = stardust
+const { Grid, Segment } = stardust
 
 class DocsApp extends Component {
   state = { menuSearch: '' };
 
   render() {
-    const components = Object.keys(stardust)
-      .sort()
-      .map(name => <ComponentDoc key={name} meta={stardust[name]._meta} />)
+    const components = _.map(stardust, '_meta')
+      .sort(({ name }) => name)
+      .map(_meta => (
```

Comment:
The stardust export has keys with deprecated access to components.  We don't show those any longer in the sidebar.